src/ert/gui/tools/workflows/run_workflow_widget.py

```diff
@@ -131,9 +131,10 @@ def startWorkflow(self) -> None:
 
         workflow_thread.start()
 
-        self._running_workflow_dialog.show()
+        dialog.show()
 
     def runWorkflow(self) -> None:
+        assert self._workflow_runner is not None
```

Comment:
Why not assert it simply? Is that not the same?

```
assert self._workflow_runner
```

Reply:
Just in case `__bool__` is overriden somewhere, I would prefer to keep it the way it is.